dbx/jvm/v5.5-wn-items.rst

```diff
@@ -0,0 +1,158 @@
+- Introduces performance improvements that are reflected in the driver performance
+  benchmarks. To view the results of the performance benchmark tests, see the following
+  :ref:`java-5.5-performance-improvements` section.
+
+.. _java-5.5-performance-improvements:
+
+Performance Improvements
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+   The tables in this section show the performance
+   results of the synchronous Java driver. The improvements
+   were made in the shared driver core and we expect similar performance
+   improvement percentages for all Java drivers.
+
+   The tests were conducted on MongoDB Server 8.0 with
+   the following configuration:
+
+   - Running on ``localhost`` with negligible network latency.
+     In higher-latency environments, you might see less significant
+     performance improvements.
+   - Authentication disabled.
+   - TLS disabled.
+   
+The following table shows the results of performance improvement 
+tests for v5.5 of the driver configured with standard transport settings:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 50 25 25
+
+   * - Feature
+     - Percent Improvement
+     - Z-score
+
+   * - Deep BSON decoding
+     - 19.44%
+     - 5.4
+
+   * - Deep BSON encoding
+     - 102%
+     - 22.8
+
+   * - Find many and empty the cursor
+     - 25.08%
+     - 13.72
+
+   * - Find one by ID
+     - 2.7%
+     - 3.16
+
+   * - Flat BSON decoding
+     - 31.2%
+     - 9.38
+
+   * - Flat BSON encoding
+     - 199.5%
+     - 12.34
+
+   * - Full BSON decoding
+     - 16.5%
+     - 7.23
+
+   * - Full BSON encoding
+     - 147.3%
+     - 10.39
+
+   * - LDJSON multi-file import
```

Comment:
Q: do we use 'LDJSON' elsewhere in the docs? don't think i've ever seen it

Reply:
Wikipedia uses JSON-LD, so I edited and linked to the wikipedia page